src/test/java/Homework20.java

```diff
@@ -0,0 +1,39 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework20 extends BaseTest{
+    @Test
+    public void deletePlaylist() {
+        logIn();
+        clickOnPlaylist();
+        clickOnDeletePlaylist();
+        assertPlaylistIsDeleted();
+
+    }
+
+    private void assertPlaylistIsDeleted() {
+        //Assert Delete Popup shows
+        WebElement playlistDeletedPopup = wait.until(ExpectedConditions.visibilityOfElementLocated
+                (By.xpath("//div[@class='success show']")));
+        Assert.assertTrue(playlistDeletedPopup.getText().contains("Deleted playlist"));
+        //Assert the playlist is no more listed in the playlists
+        Assert.assertTrue(wait.until(ExpectedConditions.invisibilityOfElementLocated
+                (By.xpath("//a[text()='Salma']"))));
+    }
+
+    private void clickOnDeletePlaylist() {
+        WebElement deletePlaylist = wait.until(ExpectedConditions.visibilityOfElementLocated
+                (By.xpath("//button[@title='Delete this playlist']")));
+        deletePlaylist.click();
+    }
+
+    private void clickOnPlaylist() {
+        WebElement playlistLocator = wait.until(ExpectedConditions.visibilityOfElementLocated
```

Comment:
you can move the wait method to the base class and then use it in other methods, so as not to write wait and etc .... every time, but simply pass xpath or css

Reply:
ok, got it